.github/workflows/test-unit.yml

```diff
@@ -45,5 +37,30 @@ jobs:
       - name: Type Check
         run: pnpm typecheck
 
+  build:
+    name: Unit Test (${{ matrix.node-name }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['16']
+        include:
+          - node-version: '16'
+            node-name: 'Latest'
```

Comment:
Perhaps we don't need to use a matrix here either because both the OS and node version have only one.

Reply:
node 18 will be lts in few months later, so it will be added later.

Reply:
Okay, got it.